.drone/drone.yml

```diff
@@ -1500,6 +1500,7 @@ name: docker-driver
 steps:
 - commands:
   - make docker-driver-push
+  - make docker-driver-push PLUGIN_ARCH=-arm64
```

Comment:
You have to change the jsonnet file uinstead. I'll generate the Drone YML then.

Reply:
done

Reply:
@jeschkies a friendly ping to let you know this PR has been rebased but requires your intervention for the `.drone/drone.yml` generation 🙂

Reply:
we don't use drone anymore